Makefile

```
@@ -78,8 +82,10 @@ lint: fmt vet
 	git diff --exit-code
 
 # Generate code
-generate: controller-gen
+generate: controller-gen autorest
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
+	$(AUTOREST) autorest --go --package-name="github.com/vshn/stardog-userrole-operator/stardogrest" --namespace="stardogrest" --input-file="stardogrest/stardog_swagger.yaml" --output-folder="stardogrest"
+	go fmt ./...
```

Thinking about this, since autorest depends on node (I assumed it's a Go binary...), I would move the last 2 commands to the `autorest` target. This target can be executed when the stardog API is updated, but it doesn't need to be done with every build.
```suggestion
generate: controller-gen
	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
```